shenyu-plugin/shenyu-plugin-rpc-context/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>shenyu-plugin</artifactId>
+        <groupId>org.apache.shenyu</groupId>
+        <version>2.4.2-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>shenyu-plugin-rpc-context</artifactId>
```

Comment:
The name rpccontext is the same as Dubbo's rpccontext, which is easy to be misunderstood

Reply:
Yes, it was originally to pass Dubbo's rpccontext, so I named the plug with rpccontext. Or you can suggest a better name to explain the meaning of the plug-in.

Reply:
Why not make a general context? The client plugin obtains and injects its own implementation from this context, and the server plugin obtains and writes the implementation of this context from its own request.After all, besides Dubbo, grpc also has its own context to pass

Reply:
Sounds great, how about rename to `shenyu-plugin-general-context`?Or maybe just `shenyu-plugin-context`?